sled-agent/src/services.rs

```diff
@@ -928,6 +950,13 @@ impl ServiceManager {
                                 "config/board_rev",
                                 &self.inner.sidecar_revision,
                             )?;
+                            smfh.setprop(
+                                "config/sidecar_uuid",
+                                &self
+                                    .inner
+                                    .sidecar_uuid
+                                    .unwrap_or_else(Uuid::new_v4),
```

Comment:
Is this right? If we don't have a UUID for the sidecar when we're launching the service, and we just make one up here, it won't be stable at all. Isn't that a problem for associating data?